### ISSUE TITLE
Missing links to navigate to the previous or next Understanding document.

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wcag#2953]
For many Understanding documents in the ["Other Understanding documents" section](https://www.w3.org/WAI/WCAG21/Understanding/#other), the links to the other documents are missing in the header; instead, the following is displayed:

> ~ Beginning of suite ~ ~ End of suite ~

**Comment by kfranqueiro** [comment]
This has been resolved by #3917 for 2.2, and will be resolved for 2.1 when we're able to re-generate with the new build system.

**Comment by kfranqueiro** [comment]
The 2.1 docs have now been republished using the new build system, so this is resolved there as well.
